guides/source/action_text_overview.md

```diff
@@ -112,6 +112,110 @@ class MessagesController < ApplicationController
 end
 ```
 
+## Rendering Rich Text content
+
+ActionText will sanitize and render rich content on your behalf.
+
+By default, the Action Text editor and content is styled by the Trix defaults.
+
+If you want to change these defaults, you'll want to remove the
+`app/assets/stylesheets/actiontext.scss` linker and base your stylings on the
```

Comment:
Instead of `linker` we could say remove `// require 'actiontext.scss'`. Sounds more readable and easy to understand. 

Also, `If you want to change these defaults` -> removing linker doesn't seem right. We can always override the current CSS. Instead we can say `If you want to remove these defaults`

Reply:
Yeah, l think we need to me more explicit what `linker` means.

Reply:
Done!